COPY/etc/httpd/conf.d/manageiq-redirects-cockpit

```diff
@@ -0,0 +1 @@
+# Managed by cockpit-ws worker when enabled
```

Comment:
Adding it here ensures apache won't error if it tries to start before the cockpit-ws worker has had a chance to do anything. Just trying to eliminate a potential race.